client-spring/src/main/java/com/netflix/conductor/client/spring/ClientProperties.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.netflix.conductor.client.spring;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+
+@ConfigurationProperties("com.netflix.conductor.client")
```

Comment:
Consider using a different prefix for the properties, maybe just ```conductor.client``` 
Using the package name as the property prefix leaks out implementation details.